x-pack/plugins/timelines/common/types/timeline/data_provider/index.ts

```diff
@@ -13,8 +13,11 @@ export const IS_OPERATOR = ':';
 /** The `exists` operator in a KQL query */
 export const EXISTS_OPERATOR = ':*';
 
+/** The `is one of` operator in a KQL query */
+export const IS_ONE_OF_OPERATOR = 'includes';
+
 /** The operator applied to a field */
-export type QueryOperator = ':' | ':*';
+export type QueryOperator = ':' | ':*' | 'includes';
```

Comment:
Don't forget to update this type if you end up updating the other one above

Reply:
changed!